ml_pipeline_engine/node/node.py

```diff
@@ -39,19 +38,12 @@ def get_node_id(node: NodeLike) -> NodeId:
     return '__'.join([node_type, node_name])
 
 
-def get_run_method(node: NodeLike) -> t.Optional[str]:
-    run_method = None
+def get_run_method(node: NodeBase) -> t.Optional[str]:
```

Comment:
This method doesn't need anymore due to the only one possible base class. Let's eliminate the method and work with the direct method "process"